phoenix-kafka-base/phoenix5-kafka/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>phoenix-kafka-base</artifactId>
+        <groupId>org.apache.phoenix</groupId>
+        <version>6.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>phoenix5-kafka</artifactId>
+    <name>Phoenix5 - Kafka</name>
+
+    <properties>
+        <top.dir>${project.basedir}/..</top.dir>
+        <phoenix.version>5.1.0-SNAPSHOT</phoenix.version>
+        <hbase.version>2.2.4</hbase.version>
+        <hadoop.version>3.0.3</hadoop.version>
+        <hadoop-two.version>3.0.3</hadoop-two.version>
+        <phoenix.compat.version>5</phoenix.compat.version>
+        <curator.version>4.0.0</curator.version>
+        <jdk.version>1.8</jdk.version>
+        <maven-shade-plugin.version>3.1.1</maven-shade-plugin.version>
+        <jetty.version>9.3.19.v20170502</jetty.version>
+    </properties>
+
+
+    <dependencies>
```

Comment:
These are additional dependencies that are not needed for 4.x ?

Reply:
Yes, they are.